File: app/src/main/java/com/example/netnew/MyApplication.java
package com.example.netnew;

import android.app.Activity;
import android.app.Application;

public class MyApplication extends Application {
    public static MyApplication Instance;
    @Override
    public void onCreate() {
        super.onCreate();
        Instance = this;
    }
    private Activity mMainActivity;

    public Activity getMainActivity() {
        return mMainActivity;
    }

    public  void setMainActivity(Activity mainActivity) {
        mMainActivity = mainActivity;
    }
}

File: app/src/main/java/com/example/netnew/MyApplication.kt
package com.example.netnew

import android.app.Application
import com.example.netnew.MyApplication
import android.app.Activity

class MyApplication : Application() {
    override fun onCreate() {
        super.onCreate()
        Instance = this
    }

    var mainActivity: Activity? = null

    companion object {
        var Instance: MyApplication? = null
    }
}